packages/paste-design-tokens/tokens/aliases/line-height.yml

```diff
@@ -1,5 +1,6 @@
 aliases:
   line-height-0: "0" # 0
+  line-height-05: "0.75rem" # 8px * 1.4 = 12px / 16 = IDK SOMETHING
```

Comment:
@SiTaggart can you help with the font size comment here

Reply:
Added suggestion. 

All the comment does is "show your workings out" in a math context.